### ISSUE TITLE
Missing floatThead.js

### ISSUE THREAD

**Issue by tomhukins** [ctrlo/GADS#106]
When I run `perl Makefile.PL` I see:
```
Checking if your kit is complete...
Warning: the following files are missing in your kit:
        public/js/jquery.floatThead-slim.min.js
Please inform the author.
```
It looks to me like d734d3947d2ada intended to replace this with the latest released version but instead removed it.

**Comment by tomhukins** [comment]
..and that same commit moved the file to public/js/jquery.floatThead.min.js but didn't update the MANIFEST.

**Comment by abeverley** [comment]
Thanks @tomhukins , fixed in a825fe31
